### PR TITLE
append timestamp, if any, to each weekday entry

### PR DESCRIPTION
**Pull request by lenient7**
When |Wkday| is used with a timestamp, append the timestamp, if any, to each weekday entry.

### CODE REVIEW

File: org-recur.el [hunk 1]
```diff
@@ -5,6 +5,8 @@
 ;; Author:      Marcin Swieczkowski <marcin.swieczkowski@gmail.com>
 ;; Created:     Fri Feb 15 2019
 ;; Version:     1.3.2
+;; Package-Version: 1.3.2
+;; Package-Commit: fa2a37d50229346929fa70b56c9d7cc34bc628ae
```
Comment (mrcnski) [inline org-recur.el:9]: I think you don't need to add these, looks like they are automatically added by the package manager. For example in the `org-recur.el` in my `elpa` folder I see:

```el
;; Package-Version: 20221229.1101
;; Package-Commit: fa2a37d50229346929fa70b56c9d7cc34bc628ae
```

**Review by mrcnski — CHANGES_REQUESTED** [review]
Thanks for the contribution! Unfortunately, it looks like it doesn't work as-is. 😅 I identified at least three issues with the code:

1. I tested with:

```el
(message (org-recur--get-next-date "|Tue,Fri| test"))
``` 

and it returned `"Tue Fri"` instead of `"Tue"`.

2. Also, what if the recurrence string is `|10:00,18:00|`? This is perfectly valid, as someone may want to do something at 10:00 and 18:00 every day, and right now it works. But with new logic, it would try to apply `18:00` to all options! 😅 

3. Also I think right now having the timestamp first e.g. `18:00 Tue` is valid, but your code seems to always look at the last position of the options entry, so it would try to apply `Tue` to all options.

## Potential Fix

Thinking out loud, I think the logic should instead apply specifically only for instances of `Wkdy`, and should happen at the same time we transform `Wkdy` (instead of after it's already been transformed). This way you can have `Sun,Wkdy 14:00` without affecting `Sun`. We also have to consider the case of `14:00 Wkdy`.

Does that make sense? I know this was just a quick hack for your purposes, so I can pick this up if you're busy. Let me know, and thanks!

**Comment by lenient7** [comment]
Thank you very much for your quick review and feedback. 

You are correct. This was a quick hack for my specific use case of "Wkdy".  Never thought about item 1. I also did not know that item 2 is possible and that having a timestamp first should work as well. 

Your suggestion makes great sense to me: 1. This hack should apply specifically only for the instances of Wkdy, and 2. the case of 14:00 Wkdy should work, too. 

You are more than welcome to pick this up if you can since I will not be able to time to work on this till weekends. :)

File: org-recur.el (hunk 1 above)
Comment (lenient7) [reply org-recur.el:9]: I think that's what my magit setup automatically added. I will remove it in the next revision. Thanks!

**Comment by mrcnski** [comment]
Great, I'll let you know if/when I start working on it!

**Comment by mrcnski** [comment]
I finally have some time; will give this a shot now.

**Comment by lenient7** [comment]
Thank you for letting me know. Hope it goes well. Let me know if I can be of help!

**Comment by mrcnski** [comment]
@lenient7 It's ready! https://github.com/mrcnski/org-recur/pull/10 Can you let me know if this works for you?

**Comment by lenient7** [comment]
I tested with my use cases and this works without an issue. Thank you very much!

**Comment by mrcnski** [comment]
Superseded by https://github.com/mrcnski/org-recur/pull/10. Thank you for the initial contribution and pushing for a fix. 💪

**Comment by mrcnski** [comment]
@lenient7 It's ready! https://github.com/mrcnski/org-recur/pull/10 Can you let me know if this works for you?

**Pull request closed without merging**